src/main/haskell/kore/test/Test/Kore/Step/Simplification/Or.hs

```diff
@@ -47,38 +51,64 @@ import qualified Test.Kore.Step.MockSymbols as Mock
 --    m or M : a character neither top nor bottom. Two values
 --             named `pm` and `pM` are expected to be unequal.
 
-test_highestTop :: TestTree
-test_highestTop =
-  testGroup "Top dominates only when all its components are top"
-  [ ((tT, pT, sT), (tm, pm, sm)) `simplifiesTo_` (tT, pT, sT)
-  , ((tm, pm, sm), (tT, pT, sT)) `simplifiesTo_` (tT, pT, sT)
-  , ((tT, pT, sT), (tT, pT, sT)) `simplifiesTo_` (tT, pT, sT)
-  ]
-
-test_mergePredicates :: TestTree
-test_mergePredicates =
-    testGroup "Merge predicates when first term is Top and substitutions are equal"
-    [ ((tT, pM, sT), (tm, pm, sT)) `simplifiesTo_` (tT, makeOrPredicate pM pm, sT)
-    -- This fails because the halfSimplifyEvaluated is not commutative:
-    -- , ((tm, pm, sT), (tT, pM, sT)) `simplifiesTo_` (tT, makeOrPredicate pm pM, sT)
-    , ((tT, pM, sm), (tm, pm, sm)) `simplifiesTo_` (tT, makeOrPredicate pM pm, sm)
-    , testGroup "Inequal substitutions prevent merging"
-        [ ((tT, pM, sT), (tm, pm, sm)) `becomes_` MultiOr [orChild (tT, pM, sT), orChild (tm, pm, sm)]
-        , ((tm, pm, sm), (tT, pM, sT)) `becomes_` MultiOr [orChild (tT, pM, sT), orChild (tm, pm, sm)]
+test_topTermAnnihilates :: TestTree
+test_topTermAnnihilates =
+    testGroup "\\top term annihilates \\or when other components are equal"
+        [ expectation
```

Comment:
It feels that understanding what is being tested here is harder than it should. I'm not sure what would be the best thing to do, but why not separate this into four tests, one for each branch of `expectation`?

Reply:
The first two branches actually are test code in another module that already has its own tests, so I think I should just remove them, and let this test be responsible only for testing code in `Kore.Step.Simplification.Or`.